Makefile

```diff
@@ -1,15 +1,15 @@
 .PHONY: build
-build:
+build: generate
 	go build -o dist/promscale ./cmd/promscale
 
-test:
+test: generate
```

Comment:
We should add `.PHONY: test` here (we're actually missing `.PHONY` on all of the phony targets, such as `all`, `generate`, `e2e-test`, `go-fmt`, etc.